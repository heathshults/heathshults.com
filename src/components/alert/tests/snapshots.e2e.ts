import { snapIt } from '../../../test';

const component = 'alert';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('without parameters', '<hs-alert>test this!</hs-alert>');
    snap('correct type', '<hs-alert type="brand">test this!</hs-alert>');
  });
});
