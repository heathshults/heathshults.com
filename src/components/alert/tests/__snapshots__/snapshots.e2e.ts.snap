// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alert renders correct type 1`] = `"<hs-alert type=\\"brand\\" class=\\"hydrated\\"><div role=\\"alert\\" hidden=\\"\\" class=\\"hs-alert hs-alert--brand u-display-none\\">test this!</div></hs-alert>"`;

exports[`alert renders without parameters 1`] = `"<hs-alert class=\\"hydrated\\"><div role=\\"alert\\" hidden=\\"\\" class=\\"hs-alert u-display-none\\">test this!</div></hs-alert>"`;
